packages/components/src/globals/scss/_theme-tokens.scss

```diff
@@ -105,13 +105,18 @@
   /// @type Number
   /// @access public
   /// @group button
-  $button-padding-ghost: 11px 13px !default !global;
+  $button-padding-ghost: calc(0.875rem - 3px) 12px !default !global;
```

Comment:
Could you add a quick comment around where the `calc` values come from and how they work?

Reply:
It mimics the y padding for the regular field token above (like the small ghost button does). The first number causes the change in height, the second number controls for the border width presumably. The goal here was just to use the same y padding calculation as the normal field token.

Reply:
Could you add this in the source code so that maintainers looking back at this will see where it comes from? That's all I meant, not that it was an invalid usage or anything. Just to help folks who refer to this understand where the numbers come from and how they're used.